### DISCUSSION TITLE
Feature Request - new item type

### DISCUSSION THREAD

**Discussion by veenone** [dani-garcia/vaultwarden#3901] [Ideas]
Hi all,

I have a request for a new feature, which is to add more item type in addition to the currently available one.
I'm managing access codes which  only requires pin (i.e access control devices, cabinet, vault,).
do you think having 1 more item with such category (i.e access codes) would be feasible? 
where the required mandatory fields are only (name, description, pin value)

thank you

**Comment by BlackDex** [comment]
Vaultwarden is only the backend part. This project does not maintain the clients which are maintained by Bitwarden.

Adding something like that would need to be done for all clients.
So, something like this would best be requested at the Bitwarden repo.

But, i think a simple login item should work just as well.

**Reply by stefan0xC** [reply]
**Custom Item Types** are also on the [Bitwarden Roadmap](https://community.bitwarden.com/t/bitwarden-roadmap/12865) (currently still under research).
[![Bitwarden Roadmap](https://community.bitwarden.com/uploads/default/original/3X/d/a/da60bd9266674861ab26b1fc1c1342ea4f074840.png)](https://community.bitwarden.com/t/bitwarden-roadmap/12865)

**Reply by veenone** [reply]
ah I see, 
Thank you for your both support by providing the explanation that I was looking for :)
